src/graphql/magentoArgsTransformer.js

```diff
@@ -0,0 +1,156 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+'use strict';
+
+/**
+ * argsTransforms defined how to transform each argument
+ * checkFields sets arguments that should be checked in any case for a certain field
+ */
+
+const { EnumType } = require('json-to-graphql-query');
+const MissingPropertyError = require('@adobe/commerce-cif-common/exception').MissingPropertyError;
+
+ /**
+ * @private
+ */
+function _translateFieldName(field) {
+    if (field.startsWith('name')) {
+        return 'name';
+    }
+    if (field.startsWith('price')) {
+        return 'price';
+    }
+    if (field.startsWith('categories.id')) {
+        return 'category_id';
+    }
+    if (field.includes('sku')) {
+        return 'sku';
+    }
+    if (field.startsWith('createdAt')) {
+        return 'created_at';
+    }
+    if (field.startsWith('lastModifiedAt')) {
+        return "updated_at";
+    }
+    return null;
+}
+
+let transformerFunctions = {
```

Comment:
Can you add a quick comment to each of those functions to describe the kind of transformation that is done?